ktor-client/ktor-client-plugins/ktor-client-websockets/common/test/io/ktor/client/tests/plugins/WebSocketRemoteTest.kt

```diff
@@ -121,21 +121,27 @@ class WebSocketRemoteTest : ClientLoader() {
     private class CustomException : Exception()
 
     @Test
-    fun testErrorHandling() = clientTests(listOf("Android", "Apache", "Curl")) {
+    fun testErrorHandling() = clientTests(skipEngines) {
         config {
             install(WebSockets)
         }
 
         test { client ->
-            assertFailsWith<CustomException> {
```

Comment:
BTW, this should work with `assertFailsWith(CustomException::class)` instead of using reified version (had the same issue in rsocket-kotlin repository